index.html

```diff
@@ -0,0 +1,124 @@
+            <li class="optionsheader2">Join</li>
+            <li class="optionsheader2">Sponsor</li>
+            <li class="optionsheader2">News</li>
+            <li class="campaign">CC Campaign</li>
+          </ul>
+        </div>
+
+        <div class="header3">
+          <p class="hello">"Hello! Sharing a world concert in Bolivia"</p>
+          <p class="massive">WELCOME TO THE MASSIVE LIVE CONCERT VERSION<br>2022</p>
+          <p class="extreme">An extreme massive live concert gathering world class artists for the rejoice of those people who loves good music and good performances transmitted to a more than 80 countries all around the world. </P>
+          <p class="date">2022.10.14(THU) ~ 15(FRI)</p>
+          <p>Estadium Hernando Siles, La Paz - Bolivia</p>
+        </div>
+      </div>
+    </div>
+  </header>
+
+  <section class="mainprogram">
+    <h1 id="mainprog" class="mainprog">Performing Acts</h1>
+    <div class="maincontain">
+      <ul class="mainreferences">
+        <li><img src="./images/opening.png" alt="Billie Eilish"></li>
+        <li class="redtext">Billie Eilish</li>
+        <li class="normaltext">An American singer and songwriter. She is best known for her successful debut single, Ocean Eyes. She belongs to a family of musicians, she was destined to make a career in performing arts.</li>
+      </ul>
+      <ul class="mainreferences">
+        <li><img src="./images/secondact.png" alt="Lecture"></li>
+        <li class="redtext">U2</li>
+        <li class="normaltext">U2, Irish postpunk band that by the end of the 1980s had established itself not only as one of the world’s most popular bands but also as one of its most innovative</li>
+      </ul>
+      <ul class="mainreferences">
+        <li><img src="./images/thirdact.png" alt="Lecture"></li>
+        <li class="redtext">Madonna</li>
+        <li class="normaltext">Madonna, American singer, songwriter, actress, and entrepreneur immensely popular in the 1980s and ’90s becoming to be the "queen of pop" until our days.</li>
+      </ul>
+      <ul class="mainreferences">
+        <li><img src="./images/fourthact.png" alt="Lecture"></li>
+        <li class="redtext">Coldplay</li>
+        <li class="normaltext">Coldplay, British rock group whose melodic piano-driven anthems lifted it to the top of the pop music world in the early 21st century, formed in 1998, and leaded by pianist-vocalist Chris Martin.</li>
+      </ul>
+      <ul class="mainreferences">
+        <li><img src="./images/fifthact.png" alt="Lecture"></li>
+        <li class="redtext">Mariah Carey</li>
+        <li class="normaltext">Mariah Carey, American pop singer, noted for her remarkable vocal range. She was one of the most successful female performers of the 1990s and remained popular into the early 21st century.</li>
+      </ul>
+    </div>
+    <p class="tickets">Where to Buy Tickets</p>
```

Comment:
- [ ] Kindly change this `p` tag to `button` or `input type="button"` for semantic purposes.
- [ ] Kindly increase the left and right spacing for that button to match the Behance design in the mobile version.

| Your design | Behance design | 
| -- | --- | 
| ![image](https://user-images.githubusercontent.com/24703171/169204292-9b6198b6-9b1f-4645-8833-1cc6ff74f70e.png) | ![image](https://user-images.githubusercontent.com/24703171/169204502-b36e0597-353c-4065-b1e6-9990ef9b1dea.png) |

- [ ] Kindly change the design of the button in the `desktop version` to match the Behance design.
![image](https://user-images.githubusercontent.com/24703171/169204856-6b880247-eda6-48c7-be34-3616270d0801.png)

Reply:
The p tag was changed as instructed. Also the spacing was modified trying to match the design.
The design button in the desktop version was corrected.
Thanks!!!